frontend/src/app/survey/detail/result-overview/result-overview.component.ts

```diff
@@ -0,0 +1,44 @@
+import {Component, Input, OnInit} from '@angular/core';
+import {TimeslotDto} from '../../../api/models/timeslot-dto';
+import {ResponseDto} from '../../../api/models/response-dto';
+
+@Component({
+    selector: 'app-result-overview',
+    templateUrl: './result-overview.component.html',
+    styleUrls: ['./result-overview.component.sass']
+})
+export class ResultOverviewComponent implements OnInit {
+
+    @Input() timeSlots: TimeslotDto[] = [];
+    @Input() responses: ResponseDto[] = [];
+    displayedColumns: string[] = [];
+
+    constructor() {
+    }
+
+    ngOnInit(): void {
+        setTimeout(() => {
+            this.displayedColumns = ['name', ...this.timeSlots.map(timeslot => timeslot.id.toString())];
+        }, 500);
+    }
+
+    responseForParticipantAndTimeslot(participant: ResponseDto, timeslot: TimeslotDto): boolean | null {
+        const result = participant.responses.find(r => r.timeslotID === timeslot.id);
+
+        return result ? result.value : null;
+    }
+
+    appendLeadingZeroes(n: number): string {
+        return n <= 9 ? `0${n}` : `${n}`;
```

Comment:
There are already build in functions for this. Something like:
`n.toString().padStart(2, '0')`.
We should not reimplement this.